index.html

```diff
@@ -111,14 +111,14 @@ <h5>CANOPY</h5>
             </div>
 
             <div class="card port-four">
-                <div class="profile order-1"><img src="image-source/Snapshoot Portfolio (3).png" alt="Snapshoot"></div>
+                <div class="profile order-1"><img src="image-source/Snapshoot Portfolio (3).png" alt="project 4"></div>
                 <div class="card-port-description order-2">
                     <h3>Multi-Post Stories</h3>
                     <div class="profile-description">
                         <h5>CANOPY</h5>
-                        <img src="image-source/Counter.png" alt="Counter">
+                        <img src="image-source/Counter.png" alt="">
                         <p>Back End Dev</p>
-                        <img src="image-source/Counter.png" alt="Counter">
+                        <img src="image-source/Counter.png" alt="">
                         <p>2015</p>
```

Comment:
- [ ] Great job on adding the alt attribute here 🎉 it really makes your page more accessible. In some cases, it's ok to leave it empty and you did that well in some others but in this case, it's necessary to tell the reader that this is a  logo of ..., in case it's not rendered. 🤓